model/cpython/metrics.yaml

```
@@ -0,0 +1,39 @@
+groups:
+  - id: metric.cpython.gc.collections
+    type: metric
+    metric_name: cpython.gc.collections
+    brief: "The number of times a generation was collected since interpreter start."
+    note: >
+      This metric reports data from [`gc.stats()`](https://docs.python.org/3/library/gc.html#gc.get_stats)
+    instrument: counter
+    unit: "{collection}"
+    stability: development
+    attributes:
+      - ref: cpython.gc.generation
+        requirement_level: required
+
+  - id: metric.cpython.gc.collected.objects
+    type: metric
+    metric_name: cpython.gc.collected.objects
```

I think this fits the "adjective_noun" guidance, see https://github.com/open-telemetry/semantic-conventions/issues/1755#issuecomment-2597358622

```suggestion
    metric_name: cpython.gc.collected_objects
```